src/rest_srr_GET.ecpp

```diff
@@ -61,27 +61,43 @@ UserInfo user;
     
     try
     {
-        // Get the feature list managed
+        // Get feature list managed
         dto::srr::SrrFeaturesListDto featuresListDto = getFeatureListManaged();
+        
+        // Output serialization
+        cxxtools::SerializationInfo siResp;
+        siResp.addMember(SRR_VERSION) <<= ACTIVE_VERSION;
 
-        // Build the json result
-        cxxtools::SerializationInfo jsonResp;
-        jsonResp.addMember(SRR_VERSION) <<= ACTIVE_VERSION;
-        cxxtools::SerializationInfo& siFeaturesList = jsonResp.addMember(FEATURE_LIST_NAME);
+        cxxtools::SerializationInfo siFeaturesList;
         siFeaturesList.setCategory(cxxtools::SerializationInfo::Category::Array);
+        siFeaturesList.setName(FEATURE_LIST_NAME);
 
-        for (const auto& featureName : featuresListDto.featuresList)
+        for (const auto& srrFeatureDto :featuresListDto.featuresList)
         {
-            cxxtools::SerializationInfo siTemp;
-            siTemp.addMember(FEATURE_NAME) <<= featureName;
-            siFeaturesList.addMember("") <<= siTemp;
+            cxxtools::SerializationInfo siFeature;
+            siFeature.addMember(FEATURE_NAME) <<= srrFeatureDto.name;
+            // Dependencies
+            cxxtools::SerializationInfo& siDependencies = siFeature.addMember(DEPENDENCIES_NAME);
+            // Dependcies array
+            cxxtools::SerializationInfo siArray;
+            siArray.setCategory(cxxtools::SerializationInfo::Category::Array);
+            // Get all dependencies splited.
+            std::vector<std::string> dependenciesList = split(srrFeatureDto.dependencies, STRING_DELIMITER);
+            for (const auto& dependency :dependenciesList)
+            {
+                siArray.addMember("") <<= dependency;
+            }
+            siDependencies <<= siArray;
+            siDependencies.setName(DEPENDENCIES_NAME);
+            siFeaturesList.addMember("") <<= siFeature;
```

Comment:
_Note for later: find a way to generate that serialization/deserialization code automatically._